pkgs/build-support/cc-wrapper/cc-wrapper.sh

```diff
@@ -1,7 +1,13 @@
-#! @shell@ -e
+#! @shell@
+set -e -o pipefail
+shopt -s nullglob
+
 path_backup="$PATH"
-if [ -n "@coreutils_bin@" ]; then
-  PATH="@coreutils_bin@/bin:@gnugrep_bin@/bin"
+
+# phase separation makes this look useless
```

Comment:
can you add some more info to this comment as to what it means, for future archeologists?